addons/patches/melb/CfgAmmo.inc

```diff
@@ -0,0 +1,34 @@
+class cfgAmmo {
+	class M_AT;
+	class SWS_Dumbfire_Base : M_AT // Same as default hydras
+	{
+    ITEM_META(0);
+		cmImmunity = 1;
+		whistleDist=24;
+		effectsMissileInit = "MissileDAR1";
+		muzzleEffect = "BIS_fnc_effectFiredRocket";
+	};
+	class SWS_M_Talon : SWS_Dumbfire_Base 
+	{
```

Comment:
Double check this + the others, I've run into issues omitting file extensions in some places before so for safety I just include them whether or not they're necessary.
```suggestion
		model = "melb\proxies\m229_fly.p3d";
```